skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -342,6 +342,7 @@ public function executeIndex( $params ) {
 				$this->pageType = wfMsg( 'oasis-page-header-subtitle-forum' );
 				break;
 		}
+		wfRunHooks( 'PageHeaderPageTypePrepared', array( &$this, $ns ) );
```

Comment:
Short array notation and all objects are passed by reference since PHP5 so you don't need the `&`. :)

I'd also consider passing a Title object from context (`$this->getContext()->getTitle()`) instead of just the namespace ID since it might be useful in more contexts.